redbot/cogs/audio/sql_statements.py

```diff
@@ -241,30 +289,59 @@
     track_info = excluded.track_info,
     last_updated = excluded.last_updated
 """
-YOUTUBE_UPDATE = """
+YOUTUBE_UPDATE: Final[
+    str
+] = """
 UPDATE youtube
 SET last_fetched=:last_fetched
 WHERE track_info=:track;
 """
-YOUTUBE_QUERY = """
+YOUTUBE_QUERY: Final[
+    str
+] = """
 SELECT youtube_url, last_updated
 FROM youtube
 WHERE
     track_info=:track
     AND last_updated > :maxage
 LIMIT 1;
 """
-YOUTUBE_DELETE_OLD_ENTRIES = """
+YOUTUBE_QUERY_ALL: Final[
+    str
+] = """
+SELECT youtube_url, last_updated
+FROM youtube
+"""
+YOUTUBE_DELETE_OLD_ENTRIES: Final[
+    str
+] = """
 DELETE FROM youtube
 WHERE
-    last_updated < :maxage;
+    last_updated < :maxage
+    LIMIT 100
```

Comment:
`LIMIT` shouldn't be specified in this query.
```suggestion
    last_updated < :maxage
```